source/slang/hlsl.meta.slang

```diff
@@ -2385,9 +2385,11 @@ extension __TextureImpl<T,Shape,isArray,0,sampleCount,0,isShadow,isCombined,form
                 // lod is not supported for 1D texture
                 if (isArray == 1)
                     // Tv read(uint coord, uint array, uint lod = 0) const
+                    //__intrinsic_asm "$0.read(uint(($1).x), uint(($1).y, uint(($1).z))";
```

Comment:
what is this comment for?

Reply:
The comment was to not delete the LOD sampling code of 1D textures.
The comment should be removed since this behavior will likely not be supported by metal in the near future.